person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] [Optional] It would be better professionally if you generate the `id` automatically, not add it as a parameter while creating a new person.

It will be like that:
```ruby
 def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
```

Reply:
Thanks.. I will apply what you suggested